lib/cog_api/fake/relay_groups.ex

```diff
@@ -52,13 +52,18 @@ defmodule CogApi.Fake.RelayGroups do
   end
   defp delete_relay_group(_), do: {:error, ["The relay group could not be deleted"]}
 
-  def add_relay(_, _, %Endpoint{token: nil}), do: Endpoint.invalid_endpoint
-  def add_relay(%{name: name}, %{relay: relay_name}, %Endpoint{token: _}) do
-    relay = Server.show_by_key(Relay, :name, relay_name)
+  def add_relays(_, _, %Endpoint{token: nil}), do: Endpoint.invalid_endpoint
+  def add_relays(%{name: name}, %{relays: relay_names}, %Endpoint{token: _}=endpoint) do
     relay_group = Server.show_by_key(RelayGroup, :name, name)
-    add_relay(relay, relay_group)
+    relay_ids = Enum.map(relay_names, &Server.show_by_key(Relay, :name, &1))
+    |> Enum.map(&Map.fetch!(&1, :id))
+    add_relays(relay_group.id, relay_ids, endpoint)
   end
-  def add_relay(id, relay_id, %Endpoint{token: _}) do
+  def add_relays(id, relay_ids, %Endpoint{token: _}=endpoint) when is_list(relay_ids) do
+    Enum.map(relay_ids, &add_relays(id, &1, endpoint))
+    |> List.last
```

Comment:
I don't remember. I'm pretty sure there was a reason, I'm just not sure if it was a good one or not.